server/graphql/v2/interface/Account.js

```diff
@@ -394,36 +393,37 @@ const accountFieldsDefinition = () => ({
         );
       }
 
-      const collectives = await sequelize.query(
-        `
-        SELECT
-          c.*, COUNT(*) OVER() AS __total_count__
-        FROM
-          "Collectives" c
-        INNER JOIN "Expenses" AS "submittedExpenses"
-          ON c."id" = "submittedExpenses"."FromCollectiveId"
-          AND "submittedExpenses"."deletedAt" IS NULL
-        INNER JOIN "VirtualCards" AS vc
-          ON "submittedExpenses"."VirtualCardId" = vc."id"
-          AND vc."deletedAt" IS NULL
-          AND (vc."data"#>>'{type}') = 'MERCHANT_LOCKED'
-        WHERE c."deletedAt" IS NULL
-          AND c."type" = 'VENDOR'
-          AND "submittedExpenses"."CollectiveId" = :collectiveId
-        GROUP BY c.id
-        LIMIT :limit
-        OFFSET :offset
-      `,
-        {
-          type: sequelize.QueryTypes.SELECT,
-          model: models.Collective,
-          mapToModel: true,
-          replacements: { collectiveId: account.id, limit: args.limit, offset: args.offset },
+      const result = await models.Collective.findAndCountAll({
+        group: 'Collective.id',
+        where: {
+          type: CollectiveTypes.VENDOR,
         },
-      );
+        include: [
+          {
+            attributes: [],
+            association: 'submittedExpenses',
+            required: true,
+            include: [
+              {
+                attributes: [],
```

Comment:
@SudharakaP This empty array is there to make sure we don't select any fields from the virtual cards (same thing is done for expenses)